src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/EmptyBlock/context/defaultContext.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/Test.java b/Test.java
+index 35c9af9..369bbd3 100644
+--- a/Test.java
++++ b/Test.java
+@@ -4,19 +4,20 @@ public class Test {
+     public void test() {
+         {
+             try {  // violation context
++
```

Comment:
I think to detect such change you need to resolve this issue first - https://github.com/checkstyle/patch-filters/issues/138. Because there will be no AST nodes for a newline.

Reply:
@baratali  this has been solved in #162 

https://github.com/checkstyle/patch-filters/blob/69caf72fd229f39702ebe39de3a7454768ece1a7/src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java#L155-L163

does this solve #138?

Reply:
yes, https://github.com/checkstyle/patch-filters/pull/162 solves this problem. But #138 should be implemented anyway, because you still collect all line numbers. Imagine a class with 1000 lines - this code will create a list with 1000 elements only for this one class.